Specification.md

```diff
@@ -40,40 +40,78 @@ seq=11                                  \
 
 ## 2. Format and Naming
 
-**Sync Interest Name**: `/<group-prefix>/v=2/<parameters-digest>`
+**State Vector Data Name**: `/<group-prefix>/v=3`
 
-The State Vector is encoded in TLV format and included in the `ApplicationParameters` of the Sync Interest.
-The State Vector SHOULD be the first TLV block in the `ApplicationParameters`.
-The Interest Lifetime for Sync Interests is 1 second.
+The State Vector is encoded in the content of a signed NDN Data packet.
 
-**Data Interest Name**: `/<node-prefix>/<group-prefix>/<seq-num>`
+**Sync Interest Name**: `/<group-prefix>/v=3/<parameters-digest>`
+
+The State Vector Data is included in the `ApplicationParameters` of the Sync Interest.
+The Interest Lifetime for the Sync Interest is 1 second.
+
+**Data Interest name**: `/<node-prefix>/<group-prefix>/t=<bootstrap-time>/58=<seq>`
```

Comment:
why `58=` instead of `seq=`? (given that you already used the mnemonic for the `t=` component)